sphinx_jupyterbook_latex/events.py

```diff
@@ -47,6 +47,10 @@ def override_latex_config(app: Sphinx, config: Config) -> None:
         """
     )
 
+    # at the moment, True means list for this config
+    if type(config["jblatex_show_tocs"]) is True:  # type: ignore[comparison-overlap]
```

Comment:
should this be checking if type is `bool`?

Reply:
```python
if type(config["jblatex_show_tocs"]) is bool and config["jblatex_show_tocs"]:
```

Reply:
done in https://github.com/executablebooks/sphinx-jupyterbook-latex/pull/64/commits/ba3b77d279ea6dc572d4dcfb818a2a54b2f08f6b